wiki/Tournaments/CES/CES2/en.md

```diff
@@ -0,0 +1,119 @@
+---
+tags:
+  - CES
+---
+
+# CES League osu! Tournament 2
+
+![CES League osu! Tournament 2 Logo](img/logo.jpg)
+
+The **CES League osu! Tournament 2** (***CES2***) is an international 1v1 double-elimination osu! tournament hosted by ![][flag_NL] [Stan](https://osu.ppy.sh/users/11212255). It features one of the largest staff team in the history of osu! tournaments. This is the second installment of the CES League osu! Tournament series.
```

Comment:
```suggestion
The **CES League osu! Tournament 2** (***CES2***) is an international 1v1 double-elimination osu! tournament hosted by ![][flag_NL] [Stan](https://osu.ppy.sh/users/11212255). This is the second installment of the CES League osu! Tournament series.
```
Don't think it's necessary since it doesn't explain the tournament very much.
*Ps i've done bigger staff composition*